setup.py

```diff
@@ -39,6 +39,7 @@
         ],
         # Data validation
         "validation": ["schematics"],
```

Comment:
+5 on that. I feel that would be much better. Since we already have warnings in place for them in the documentation.
The only problem being that we still need these in the setup.py for Travis to work properly. Tox uses these to install dependencies and run the build.

Reply:
We can for their install for tests in tox.ini, like we do in Scrapy for its optional dependencies (e.g. robots.txt parsers).

Reply:
Roger that, give it a look now.